lib/OctokitAction.ts

```diff
@@ -80,6 +80,37 @@ export abstract class OctokitAction extends Action {
     }
   }
 
+  protected async findEmail(login: string): Promise<string> {
+    const identities = await this.findExternalIdentities(login);
+    if (identities.length === 0) {
+      this.log(`No SAML identity found for ${login}`);
+      return null;
+    }
+    return identities[0].samlIdentity.nameId;
```

Comment:
This is not querying user emails per say. That is impossible because such an API doesn't exist. You can only get your own emails, and that is not useful.

This is querying SSO identity of a GH user that is authorize against JumpCloud. And there exists only a single identity to work with. 
If there would be some external address, I don't really care as long as the same one is in Jira.
If there would be some other mess, the next step of finding Jira account would not pair it up. And that's fine.